var/spack/repos/builtin/packages/gnutls/package.py

```diff
@@ -84,6 +88,11 @@ def configure_args(self):
         else:
             args.append("--without-zlib")
 
+        if spec.satisfies("+brotli"):
+            args.append("--with-brotli")
+        else:
+            args.append("--without-brotli")
+
```

Comment:
```suggestion
        args += self.with_or_without("brotli")
```

I think we should be able to clean this up a bit and use `with_or_without()` here.

Reply:
If I fix this one I'll just fix the whole surroundings too.

Reply:
Up to you! It'd be nice, but I understand if that's out of scope of this PR.

Reply:
I see that this has been done.